### PR TITLE
Add Malt icon

### LINKED ISSUE #6361: Request: Malt
**Brand Name:** Malt
**Website:** https://www.malt.fr/
**Alexa rank:** 131,910

**Official resources for icon and color:** https://newsroom.malt.com/media-kit

### PR DESCRIPTION
**Pull request by Ash-Crow**
malt

**Issue:** closes #6361

**Similarweb rank:** 43,313

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
- Icon taken from the previous PR (#6362 ), I just removed the wordmark, adapted the viewport and made the changes asked in the contribution guidelines.
- Color taken directly from the official logo.

### CODE REVIEW

**Comment by Ash-Crow** [comment]
I see that some tests have failed but I don't know how to fix the svg file so that they pass.

File: icons/malt.svg [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Malt</title><path d="M12.03 0c-1.978 0-2.72 1.18-2.975 2.488.184.16.367.32.555.507l2.424 2.42 2.464-2.464c.164-.164.335-.32.51-.47C14.75 1.175 14.008 0 12.034 0Zm6.467 2.528c-1.116.008-2.157.75-2.91 1.503L4.143 15.475c-1.212 1.212-2.48 3.054-.574 4.96 1.906 1.906 3.748.638 4.956-.574L19.969 8.417c1.212-1.208 2.384-3.146.574-4.956-.678-.678-1.376-.937-2.046-.933Zm-12.839 0c-.67-.008-1.376.263-2.09.977C2.17 4.9 2.48 6.265 3.223 7.369c.223-.016 6.89-.04 6.89-.04l.837-.83-2.424-2.424c-.758-.758-1.758-1.535-2.87-1.547ZM3.912 8.864C2.197 8.864 0 9.402 0 11.962c0 1.914 1.224 2.7 2.556 2.975.16-.184 6.025-6.073 6.025-6.073Zm17.604.12c-.148.175-6.029 6.084-6.029 6.084h4.61c1.714 0 3.911-.406 3.911-3.106 0-1.974-1.18-2.72-2.488-2.974zm-.678 7.584c-.243.015-6.886.031-6.886.031l-.838.838 2.469 2.464c1.212 1.212 3.15 2.384 4.96.574 1.352-1.356 1.037-2.771.291-3.907zm-8.804 1.953-2.424 2.42c-.176.177-.359.348-.547.511.28 1.332 1.06 2.548 2.967 2.548 1.914 0 2.7-1.224 2.974-2.556-.171-.147-.338-.291-.506-.458z"/></svg>
```
Comment (PratyushJayachandran) [inline icons/malt.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Malt</title><path d="M20.538 3.463c-1.811-1.812-3.747-.64-4.958.572L4.14 15.476c-1.211 1.211-2.479 3.051-.572 4.957 1.906 1.908 3.746.64 4.957-.571L19.966 8.42c1.211-1.211 2.384-3.147.572-4.958zm-10.93-.469 2.423 2.423 2.465-2.466c.168-.168.338-.324.509-.47C14.747 1.178 14.003 0 12.029 0c-1.977 0-2.72 1.182-2.976 2.488.184.16.368.32.555.506zm4.887 17.995-2.464-2.465-2.422 2.421a9.462 9.462 0 0 1-.549.51C9.338 22.783 10.124 24 12.03 24c1.91 0 2.695-1.223 2.971-2.555-.17-.146-.339-.29-.506-.456zM8.58 8.867H3.91c-1.712 0-3.91.54-3.91 3.101 0 1.912 1.223 2.697 2.556 2.972.158-.182 6.024-6.073 6.024-6.073zm12.933.124c-.148.172-6.027 6.078-6.027 6.078h4.604c1.712 0 3.91-.404 3.91-3.1 0-1.978-1.182-2.72-2.487-2.978zM10.11 7.334l.834-.834-2.42-2.422C7.312 2.868 5.472 1.6 3.565 3.506 2.168 4.904 2.48 6.266 3.22 7.37c.226-.017 6.89-.036 6.89-.036zm3.839 9.268-.837.837 2.466 2.466c1.211 1.211 3.146 2.383 4.958.572 1.351-1.352 1.041-2.772.294-3.909-.24.017-6.881.034-6.881.034z"/></svg>
```
This should fix the issues.

**Review by PratyushJayachandran — CHANGES_REQUESTED** [review]
You may apply the suggestion to fix the issues.

**Comment by Ash-Crow** [comment]
> You may apply the suggestion to fix the issues.

Thanks! It seems like it worked. Can you please explain what was the problem and how you fixed it?

**Comment by adamrusted** [comment]
Given no feedback, happy for you to make a call on this, @Ash-Crow.
We can either add the full logo with the wordmark, contact them directly and ask for permission to just user the icon, or shut this PR down and I'll mark the issue as `won't add`.

**Comment by PeterShaggyNoble** [comment]
I agree with @adamrusted's interpretation of their guidelines; that we _can't_ use the icon on its own. This is backed up by the fact that they only make the full logo available for download in their media kit. I've always been of the opinion that, as an _icon_ library, we shouldn't have any logos in our collection. But I'm in the minority on that and the precedent has long since been set! 😆

@Ash-Crow, could you update this PR to use the full logo, please?

**Comment by PeterShaggyNoble** [comment]
Are you still working on this one, @Ash-Crow?

**Comment by Ash-Crow** [comment]
The previous task had been closed because the contributor used the full wordmark and was asked to just put the logo...

I'll try to contact Malt first, if that's ok with you.

**Comment by PeterShaggyNoble** [comment]
Have you has a response from them, @Ash-Crow?

**Review by service-paradis — APPROVED** [review]

**Pull request merged**